src/modules/services/userServices.js

```diff
@@ -1,16 +1,17 @@
-import { SHOWS_API, INVOLVEMENT_API } from './rootEndpoints'
+import { SHOWS_API, INVOLVEMENT_API, COMMENTS, APP_ID } from './rootEndpoints'
 import { get, post } from './fetchApi'
-import showMovies from '../movies.js'
-import showComments from '../comments'
 
 export const fetchShows = async () => {
-  const data = await get(SHOWS_API)
-  showMovies(data)
-  showComments(data)
+	const shows = await get(SHOWS_API)
+	return shows
 }
 
+export const postComments = async data => {
+	const result = await post(`${INVOLVEMENT_API}/${APP_ID}/${COMMENTS}`, data)
+	return result
+}
```

Comment:
[OPTIONAL]

- Good job using es6 `async/await` feature 👍 . I will strongly recommend you always use `try/catch` with `async/await `so that it will be easier for you to handle errors resulting from the API calls 🤓